app/src/main/res/values/strings.xml

```diff
@@ -506,4 +505,14 @@
         <item quantity="other">%d seconds</item>
     </plurals>
 
+    <string name="button_continue">Continue</string>
+    <string name="button_report">@string/action_report</string>
```

Comment:
this is always confusing translators, please use `@string/action_report` directly

Reply:
Done